karate-core/README.md

```diff
@@ -146,6 +146,13 @@
     | <a href="#screenshotfull"><code>screenshotFull()</code></a>
   </td> 
 </tr>
+<tr>
+  <th>Appium</th>
+  <td>
+      <a href="#screen-recording">screen recording</a>
```

Comment:
title case, `Screen Recording`

Reply:
just minor changes, great you have been tracking the new changes such as `Embed.forVideoFile()` we may need to unify video recording somehow in the future but for now let us experiment

Reply:
changed title case. yes, something thing like a `karate.configure` that should handle this internally. for both mobile and web.